.github/workflows/check_pr.yml

```diff
@@ -118,10 +118,19 @@ jobs:
         with:
           node-version: 18
 
-      - name: Install dependencies
+      - name: Cache node modules
+        id: cache
+        uses: actions/cache@v3
+        with:
+          path: |
+            **/docs/node_modules
+          key: node_modules-${{ matrix.os }}-${{ matrix.node }}-${{ hashFiles('**/npm-shrinkwrap.json') }}
```

Comment:
Let's check that we're pointing at the right file here and that `**/npm-shrinkwrap.json` is not pointing at the CLI folder rather than the docs

Reply:
Did a few tests and now everything should be pointing properly to the docs library where the package files are available.